go.mod

```
@@ -17,22 +17,23 @@ require (
 	github.com/onsi/ginkgo v1.16.5
 	github.com/onsi/gomega v1.27.8
 	github.com/pborman/uuid v1.2.1
-	github.com/prometheus/client_model v0.3.0
-	github.com/prometheus/common v0.37.0
-	github.com/prometheus/procfs v0.8.0
+	github.com/prometheus/client_model v0.4.0
+	github.com/prometheus/common v0.44.0
+	github.com/prometheus/procfs v0.10.1
 	github.com/shirou/gopsutil v3.21.11+incompatible
 	github.com/spf13/pflag v1.0.5
 	github.com/stretchr/testify v1.8.4
 	go.opencensus.io v0.24.0
-	golang.org/x/crypto v0.10.0
+	golang.org/x/crypto v0.11.0
 	golang.org/x/oauth2 v0.9.0
-	golang.org/x/sys v0.9.0
+	golang.org/x/sys v0.10.0
 	google.golang.org/api v0.114.0
-	k8s.io/api v0.17.17
-	k8s.io/apimachinery v0.17.17
-	k8s.io/client-go v11.0.1-0.20190805182717-6502b5e7b1b5+incompatible
-	k8s.io/component-base v0.17.2
+	k8s.io/api v0.28.1
+	k8s.io/apimachinery v0.28.1
+	k8s.io/client-go v9.0.0+incompatible
```

Shouldn't this also `v0.28.1`?
```suggestion
	k8s.io/client-go v0.28.1
```

---

This is one is replaced with v0.28.1. please see the last line. It is related to cadvisor. I will try to upgrade that part in following pr.

---

Ah, ok. Thanks for the explanation.

---

Pretty strange to depend on cadvisor just for a log tailing lib 😄

---

It has not changed for a long time..